isl_arg.c

```diff
@@ -15,9 +15,9 @@
 #include <isl/ctx.h>
 #include <isl_config.h>
 
-static struct isl_arg help_arg[] = {
-ISL_ARG_PHANTOM_BOOL('h', "help", NULL, "print this help, then exit")
-};
+static struct isl_arg *help_arg; //= {
+//ISL_ARG_PHANTOM_BOOL('h', "help", NULL, "print this help, then exit")
+//};
```

Comment:
I could not for the life of me get MSVC to accept this macro, so deleting was easier, and works for the purposes on IslPy.